src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/QuestionPriceArea.java

```diff
@@ -0,0 +1,106 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.Range;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+
+public class QuestionPriceArea {
+  int price;
+  int surface;
```

Comment:
private

Reply:
Fix in 7c916915ee4778efa9494da1165860554ec313bc